transport/src/main/java/io/netty/channel/SingleThreadEventLoop.java

```diff
@@ -89,11 +103,81 @@ public ChannelFuture register(final Channel channel, final ChannelPromise promis
         return promise;
     }
 
+    /**
+     * Adds a task to be run once at the end of next (or current) {@code eventloop} iteration.
+     *
+     * @param task to be added.
+     */
+    final void onEventLoopIteration(Runnable task) {
```

Comment:
This method seems to indicate that the `task` should be run at the end of the event loop iteration and is the counterpart to `execute` ... would `executeAfterEventLoopIteration` be more clear?